apps/files/src/components/Collaborators/EditCollaborator.vue

```diff
@@ -82,8 +101,28 @@ export default {
         // if the role has changed, always return true. The user doesn't need to understand if two bitmasks of different roles are the same!
         return true
       }
+
+      // FIXME: Datepicker is not displaying correct timezone so for now we add it manually
+      const originalExpirationDate = moment(this.originalExpirationDate).add(moment().utcOffset(), 'm').toISOString()
```

Comment:
should we use a helper function for this hack ? this is the second time this appears

Reply:
I'd prefer not to introduce workarounds as helpers since then it feels a bit like a justification of them. While this way it will be at least a nice reminder that we need to fix it. But it's more about the feeling here than real reasoning. I'd be okay in the end with creating it.